app/views/videos/show.html.haml

```diff
@@ -11,3 +11,36 @@
         .actions
           %a.btn.btn-primary(href="") Watch Now
           %a.btn.btn-default(href="") + My Queue
+
+%section.reviews.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      = form_for [@video, Review.new] do |f|
+        -token_tag
```

Comment:
Shouldn't this be `=`? Also, you shouldn't need an explicit token tag since you get one free from `form_for`. I think that is why you didn't notice the typo. (your tag is not actually being printed)